manifest.yaml

```diff
@@ -0,0 +1,37 @@
+name: repokid
+description: AWS Least Privilege for Distributed, High-Velocity Deployment.
+repository: git@github.com:lyft/repokid.git
+image_type: library
+languages:
+  - python
+builder:
+  name: s2i
+  params:
+    builder: lyft/s2ipythonlibrary:c421d769bbd8479005c5a72d3c46b1eac64cc89d
+ tests:
+  - name: unit
+    type: jailed
+    command: make test
+  - name: lint
+    type: jailed
+    command: flake8 .
+ python:
+  mypy:
+    python_versions:
+      - '3.6'
+ py3k:
+  linters:
+    - modernize
+    - pylint
```

Comment:
remove this section and the one above it, it's very unlikely this third party repo will pass our mypy / modernize checks